docs/usage.rst

```diff
@@ -572,6 +572,16 @@ If a file is added to the CMake build system by updating one of the ``CMakeLists
 will not explicitly reconfigure the project. Instead, the generated build-system will automatically
 detect the change and reconfigure the project after :meth:`skbuild.cmaker.CMaker.make` is called.
 
+Controlling exported symbol visibility
+--------------------------------------
+
+When using a GNU based compiler on Linux, scikit-build will reduce the binary size by removing all the native exported methods.
+By default, only the ``PyInit_<moduleName>`` (or ``init<moduleName>``)entry point is exported from the native module.
+If you need to preserve native exports in your module, you may set ``SKBUILD_GNU_SKIP_LOCAL_SYMBOL_EXPORT_OVERRIDE`` as a
+`CMake option <https://scikit-build.readthedocs.io/en/latest/usage.html#cmake-configure-options>`_
+or
+`setup.py option <https://scikit-build.readthedocs.io/en/latest/usage.html#scikit-build-options>`_.
+
```

Comment:
```suggestion

Note that for Python >= 3.9, setting the option `SKBUILD_GNU_SKIP_LOCAL_SYMBOL_EXPORT_OVERRIDE` to `ON` allow to ...
```

It would be nice to further improve to provide some historical contest based on the summary available at SKBUILD_GNU_SKIP_LOCAL_SYMBOL_EXPORT_OVERRIDE

Reply:
I added a little more detail as to how this works and on what systems.